### PR TITLE
feat(networks): add Safe Zodiac modules deployments

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This adds info about Safe modules for cross-chain governance into the networks package

How it works 

- `scripts/deploy_multisig_mods` : a parser that will show command to deploy the modules (deployment has to happen from module repo)
- `yarn hardhat safe:submit --network goerli --proposal proposals/enableModule.js` : enable the deployed modules in the multisig

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/types/.eslintrc.js
Comment (clemsos) [inline packages/types/.eslintrc.js:1]: that file was needed for the linter to pass

File: packages/networks/src/networks/arbitrum.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ export const arbitrum: NetworkConfig = {
   blockScan: {
     url: (address: string) => `https://blockscan.com/address/${address}`,
   },
+  bridge: {
```
Comment (clemsos) [inline packages/networks/src/networks/arbitrum.ts:7]: Rename to ˋgovernanceBridge` following comment in #12216

**Review by julien51 — APPROVED** [review]

File: packages/types/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: ['@unlock-protocol/eslint-config'],
+  plugins: ['sort-keys-fix'],
```
Comment (clemsos) [inline packages/types/.eslintrc.js:3]:
```suggestion
```

File: smart-contracts/scripts/deploy_multisig_mods
Comment (clemsos) [inline smart-contracts/scripts/deploy_multisig_mods:1]: This should only be useful once when we add modules to a safe on a network

**Pull request merged**